srsue/src/main.cc

```diff
@@ -418,6 +418,10 @@ static int parse_args(all_args_t* args, int argc, char* argv[])
      bpo::value<int>(&args->phy.force_N_id_2)->default_value(-1),
      "Force using a specific PSS (set to -1 to allow all PSSs).")
 
+    ("phy.force_N_id_1",
+     bpo::value<int>(&args->phy.force_N_id_1)->default_value(-1),
```

Comment:
could you add this to the `ue.conf.example` file? I think also `force_N_id_2` is missing there. So would be good if you could add that too.